### PR TITLE
Update lib. Fix select image SDK 33

### PR DESCRIPTION
**Pull request by lUjek-Sh**

### CODE REVIEW

**Review by Samuel-Unknown — COMMENTED** [review]
Thank you for your work! 👍

File: buildSrc/src/main/kotlin/Versions.kt [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
+        val compileSdk = 34
+        val targetSdk = 34
         val minSdk = 21
     }
 
     object Library {
         private val major = 1 // change when you make incompatible API changes
         private val minor = 6 // change when you add functionality in a backwards-compatible manner
-        private val build = 0 // change when you make backwards-compatible bug fixes
+        private val build = 1 // change when you make backwards-compatible bug fixes
```
[the remark this answers is not in the document]
Comment (lUjek-Sh) [reply buildSrc/src/main/kotlin/Versions.kt:16]: Ok

**Review by Samuel-Unknown — APPROVED** [review]

**Comment by lUjek-Sh** [comment]
When this version available in maven for use in my project?

**Comment by Samuel-Unknown** [comment]
In a few days I think, I want to fix one issue before (there is a bug with items selection when you change the folder)

**Pull request merged**